Xamarin.PropertyEditing.Mac/Controls/Custom/NumericSpinEditor.cs

```diff
@@ -0,0 +1,287 @@
+﻿using System;
+using AppKit;
+using CoreGraphics;
+using Foundation;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public class NumericSpinEditor : NSView
+	{
+		protected NumericTextField numericEditor;
+		protected NSStepper stepper;
+		protected bool editing;
+
+
+		public event EventHandler ValueChanged;
+		public event EventHandler StepperValueChanged;
+		public event EventHandler FieldEditingEnded;
+
+		public ValidationType NumericMode {
+			get { return numericEditor.NumericMode; }
+			set {
+				numericEditor.NumericMode = value;
+				Reset ();
+			}
+		}
+
+		public string StringValue {
+			get { return numericEditor.StringValue; }
+			set { numericEditor.StringValue = value; }
```

Comment:
We should not have a public setter that *only* updates the string value. Setting the string value here does not update the value stored in `stepper`!